<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gesti칩n de Animales</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="crearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear Animal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="volverButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: lightgray;
    color: black;
    border: 2px solid gray;
    border-radius: 10px;
    padding: 5px;
}

QPushButton:hover {
    background-color: silver;
}

QPushButton:pressed {
    background-color: darkgray;
}</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="informationScroll">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>571</width>
      <height>561</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>569</width>
       <height>559</height>
      </rect>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>561</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="rows">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="filterLayout">
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Filtrar por:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="speciesLabel">
       <property name="text">
        <string>Especie:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="speciesFilterInput">
       <property name="placeholderText">
        <string>Buscar por due침o</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ownerLabel">
       <property name="text">
        <string>Due침o:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ownerFilterInput">
       <property name="placeholderText">
        <string>Buscar por due침o</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameFilterInput">
       <property name="placeholderText">
        <string>Buscar por nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearFiltersButton">
       <property name="text">
        <string>Limpiar filtros</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
